### PR TITLE
[PP][OPS-5778] start page content changes

### PR DESCRIPTION
**Pull request by pawelpanasewicz**
Signed-off-by: Pawel Panasewicz <2970040+pawelpanasewicz@users.noreply.github.com>

### CODE REVIEW

**Review by Akure4Life — APPROVED** [review]

File: app/ssttpeligibility/EligibilityServiceConfig.scala [hunk 1]
```diff
@@ -20,17 +20,18 @@ import javax.inject.Inject
...
   @Inject def this(servicesConfig: ServicesConfig) = this(
     insignificantDebtUpperLimit                      = servicesConfig.getInt("eligibility.insignificantDebtUpperLimit"),
-    maximumDebtForSelfServe                          = servicesConfig.getInt("eligibility.maximumDebtForSelfServe"),
     numberOfDaysAfterDueDateForDebtToBeConsideredOld = servicesConfig.getInt("eligibility.numberOfDaysAfterDueDateForDebtToBeConsideredOld"),
     returnHistoryYearsRequired                       = servicesConfig.getInt("eligibility.returnHistoryYearsRequired"),
     taxYearEndMonthOfYear                            = servicesConfig.getInt("eligibility.taxYearEndMonthOfYear"),
     taxYearEndDayOfMonth                             = servicesConfig.getInt("eligibility.taxYearEndDayOfMonth")
   )
+
+  val maximumDebtForSelfServe: Int = 30000
```
Comment (johnrevill) [inline app/ssttpeligibility/EligibilityServiceConfig.scala:36]: Possible debugging - was this meant to be hard coded by this ticket?

File: app/ssttpeligibility/EligibilityServiceConfig.scala [hunk 2]
```diff
@@ -20,17 +20,18 @@ import javax.inject.Inject
...
 final case class EligibilityServiceConfig(insignificantDebtUpperLimit:                      Int,
-                                          maximumDebtForSelfServe:                          Int,
                                           numberOfDaysAfterDueDateForDebtToBeConsideredOld: Int,
                                           returnHistoryYearsRequired:                       Int,
                                           taxYearEndMonthOfYear:                            Int,
                                           taxYearEndDayOfMonth:                             Int) {
+
   @Inject def this(servicesConfig: ServicesConfig) = this(
     insignificantDebtUpperLimit                      = servicesConfig.getInt("eligibility.insignificantDebtUpperLimit"),
-    maximumDebtForSelfServe                          = servicesConfig.getInt("eligibility.maximumDebtForSelfServe"),
```
Comment (johnrevill) [inline app/ssttpeligibility/EligibilityServiceConfig.scala:30]: Possible debugging - was this meant to be removed by this ticket?

File: conf/application.conf [hunk 3]
```diff
@@ -196,7 +196,6 @@ assets {
 
 eligibility {
   insignificantDebtUpperLimit = 32
-  maximumDebtForSelfServe = 10000
```
Comment (johnrevill) [inline conf/application.conf:199]: Same query here too

File: app/ssttpeligibility/EligibilityServiceConfig.scala (hunk 2 above)
Comment (pawelpanasewicz) [reply app/ssttpeligibility/EligibilityServiceConfig.scala:30]: nope

File: conf/application.conf (hunk 3 above)
Comment (pawelpanasewicz) [reply conf/application.conf:199]: nope

**Pull request merged**